t/14-isotovideo.t

```diff
@@ -186,6 +186,21 @@ subtest 'error handling when loading test schedule' => sub {
     };
 };
 
+subtest 'load test success when there is no PRODUCTDIR' => sub {
```

Comment:
is this title still relevant? Or should it be 'relative casedir with relative productdir'?

Reply:
fixed